### ISSUE TITLE
publish for Scala 2.13.0-M5?

### ISSUE THREAD

**Issue by SethTisue** [monix/minitest#23]
I think your dependencies are available for M4 now? (not 100% sure)

in the Scala 2.13 community build, at e.g. https://scala-ci.typesafe.com/job/scala-2.13.x-integrate-community-build/1228/consoleFull, I'm seeing that getting minitest onto the new collections would also help unlock a bunch of downstream projects. (most of which need cats too, but cats isn't too far from being able to publish, too.)

```
[info] Project ammonite--------------------: DID NOT RUN (stuck on broken dependencies: minitest, scala-async, spray-json, tut, utest)
[info] Project better-monadic-for----------: DID NOT RUN (stuck on broken dependencies: cats, minitest, pcplod)
[info] Project eff-------------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest)
[info] Project fs2-reactive-streams--------: DID NOT RUN (stuck on broken dependencies: cats, minitest, scodec-bits, utest)
[info] Project giter8----------------------: DID NOT RUN (stuck on broken dependencies: cats, grizzled, minitest, scalamock, utest)
[info] Project http4s----------------------: DID NOT RUN (stuck on broken dependencies: cats, http4s-parboiled2, log4s, minitest, scodec-bits, spray-json, twirl, utest)
[info] Project kxbmap-configs--------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, utest)
[info] Project minitest--------------------: FAILED (RuntimeException: )
[info] Project monix-----------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest)
[info] Project sbinary---------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, utest)
[info] Project sbt-------------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, spray-json, utest)
[info] Project sbt-io----------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, utest)
[info] Project sbt-librarymanagement-------: DID NOT RUN (stuck on broken dependencies: cats, minitest, spray-json, utest)
[info] Project sbt-util--------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, spray-json, utest)
[info] Project scala-debugger--------------: DID NOT RUN (stuck on broken dependencies: minitest, scala-async, scalamock, scallop, spray-json, tut, utest)
[info] Project scala-ssh-------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, utest)
[info] Project scalafmt--------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, utest)
[info] Project scalaprops------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, utest)
[info] Project scopt-----------------------: DID NOT RUN (stuck on broken dependencies: minitest, utest)
[info] Project sttp------------------------: DID NOT RUN (stuck on broken dependencies: akka-actor, cats, minitest, scodec-bits, spray-json)
[info] Project zinc------------------------: DID NOT RUN (stuck on broken dependencies: cats, minitest, spray-json, utest)
```

**Comment by xuwei-k** [comment]
https://github.com/monix/minitest/pull/19

```
[minitest] [error] 'by-name-right-associative' is not a valid choice for '-Xlint'
[minitest] [error] 'unsound-match' is not a valid choice for '-Xlint'
```

**Comment by SethTisue** [comment]
after #19 was merged, I tried the Scala 2.13 community build and got:

```
[minitest] [error] /Users/tisue/community.213/target-0.9.12/project-builds/minitest-140990deb8a820140508cf1c8a3f44a72fc477aa/shared/src/main/scala/minitest/runner/Framework.scala:30:5: trait Runner is abstract; cannot be instantiated
[minitest] [error]     new Runner(args, remoteArgs, testClassLoader)
[minitest] [error]     ^
[minitest] [info] No documentation generated with unsuccessful compiler run
[minitest] [error] one error found
[minitest] [error] /Users/tisue/community.213/target-0.9.12/project-builds/minitest-140990deb8a820140508cf1c8a3f44a72fc477aa/shared/src/main/scala/minitest/runner/Framework.scala:30:5: trait Runner is abstract; cannot be instantiated
[minitest] [error]     new Runner(args, remoteArgs, testClassLoader)
[minitest] [error]     ^
```

this looks a consequence of https://github.com/scala/scala/pull/6589; `import sbt.testing.{Framework => BaseFramework, _}` is importing `Runner` and then (after PR 6589) that shadows `minitest.runner.Runner`

**Comment by SethTisue** [comment]
if I run the community build with `version=2.13.0-M4` I get:

```
[info] Project minitest--------------------: SUCCESS (project rebuilt ok)
```

so that's promising.

**Comment by SethTisue** [comment]
minitest remains green in the 2.13 community build. I've updated this ticket name now that Scala 2.13.0-M5 is out

**Comment by SethTisue** [comment]
scalacheck needs to publish first, I think. https://github.com/rickynils/scalacheck/issues/418

**Comment by SethTisue** [comment]
ScalaCheck 1.14.0 is now published for M5 (both JVM & JS), so this should be unblocked now, I think?

**Comment by ashawley** [comment]
@SethTisue Can you review #30?

**Comment by SethTisue** [comment]
@ashawley done

**Comment by SethTisue** [comment]
ticket is closed, but I don't see artifacts at https://repo1.maven.org/maven2/io/monix/minitest_2.13.0-M5 ?

**Comment by alexandru** [comment]
@SethTisue I don’t remember closing it, however I’ll publish it tomorrow.

**Comment by alexandru** [comment]
@SethTisue I published it with version `2.2.2`

**Comment by dejan2609** [comment]
Sorry for additional noise (spamming), I should have known better :worried:
